src/components/layer-manager/layer-item/layer-item.spec.ts

```diff
@@ -21,6 +21,8 @@ describe('LayerItem', () => {
         layer: layerMock,
         draggableClassName: 'classnamedragg',
         isOpen: false,
+        isLayerComporatorOpen: false,
+        displayLayerComporatorOpen: false,
```

Comment:
Really not important, just a spelling typo 'a' instead of 'o':
```suggestion
        isLayerComparatorOpen: false,
        displayLayerComparatorOpen: false,
```